homework5/task1/main.c

```diff
@@ -0,0 +1,61 @@
+#include "module.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+int test(void)
+{
+    int exitCode = 0;
+    int result = 0;
+    result = findMostFrequent("test1.txt", &exitCode);
+    if (result != 4 || exitCode != 0)
+    {
+        return -1;
+    }
+    result = findMostFrequent("test2.txt", &exitCode);
+    if (result != -3 || exitCode != 0)
+    {
+        return -2;
+    }
+    result = findMostFrequent("test3.txt", &exitCode);
+    if (result != 7 || exitCode != 0)
+    {
+        return -3;
+    }
+    result = findMostFrequent("test4.txt", &exitCode);
+    if ((result != 1 && result != 2) || exitCode != 0)
+    {
+        return -4;
+    }
+    result = findMostFrequent("asfawfawfewaweafev.txt", &exitCode);
+    if (exitCode != -1)
+    {
+        return exitCode;
+    }
+    return 0;
+}
+
+int main()
+{
+    if (test() != 0)
+    {
+        return -1;
+    }
+
+    printf("Enter the name of file:\n");
+    char name[100];
```

Comment:
```suggestion
    char name[100] = { '\0' };
```
Иначе это неинициализированная переменная, что запрещено стайлгайдом